batch2/deployment.yaml

```diff
@@ -49,18 +49,6 @@ spec:
            value: "{{ global.zone }}"
          - name: KUBERNETES_SERVER_URL
            value: "{{ global.k8s_server_url }}"
-{% if not deploy %}
-         - name: WORKER_TYPE
-           value: "standard"
```

Comment:
I think I also added some of these to the configuration for `batch2` in addition to `batch2-driver`

Reply:
Thanks!